Dockerfile

```diff
@@ -60,8 +60,6 @@ RUN echo "deb [signed-by=/usr/share/keyrings/hathitrust-archive-keyring.gpg] htt
 
 RUN apt-get update && apt-get install -y grokj2k-tools
 
-RUN cpan -f -i Net::AMQP::RabbitMQ
```

Comment:
`Net::AMQP::RabbitMQ` was already included by `Makefile.pl`, which is the proper place to put required modules, so I removed it.